backend/gateway/mux/mux.go

```diff
@@ -35,14 +37,21 @@ type assetHandler struct {
 	next       http.Handler
 	fileSystem http.FileSystem
 	fileServer http.Handler
+	logger     *zap.Logger
 }
 
-func copyHTTPResponse(resp *http.Response, w http.ResponseWriter) {
+func copyHTTPResponse(resp *http.Response, w http.ResponseWriter, logger *zap.Logger) {
 	for key, values := range resp.Header {
 		for _, val := range values {
 			w.Header().Add(key, val)
 		}
 	}
+	if resp.StatusCode >= 500 && resp.StatusCode <= 599 {
+		body, _ := ioutil.ReadAll(resp.Body)
+		if len(body) > 0 {
+			logger.Error("HTTP 5xx error:", zap.Int("status code", resp.StatusCode), zap.String("response body", string(body)))
```

Comment:
left the body as a string for now since we do not have an expected response struct defined for us to call json.Unmarshal on